File: back-end/GeekSpot.API/Models/UsuarioTipo.cs
using System.ComponentModel.DataAnnotations;

namespace GeekSpot.API.Models
{
    public class UsuarioTipo
    {
        [Key]
        public int UsuarioTipoId { get; set; }
        public string? Tipo { get; set; }
        public string? Descricao { get; set; }
        public int IsAtivo { get; set; }
        public DateTime DataRegistro { get; set; }
    }
}

File: front-end/utils/types/usuarioTipo.ts
export default interface iUsuarioTipo {
    usuarioTipoId: number;
    tipo: string;
    descricao: string | null;
    isAtivo: number;
    dataRegistro: Date;
    erro: boolean;
    codigoErro: number | null;
    mensagemErro: string | null;
}

